### DISCUSSION TITLE
can i change my github-name - on the fly - is this doable!?

### DISCUSSION THREAD

**Discussion by fsj-digital** [community/community#53422] [New to GitHub]
can i change my github-name - on the fly - is this doable!?

well am i able to swith the name of the gitub-account. Is this possible?!

btw: where can i check for names  - and wether they are free and available or not!?

thanks for any and all help

**Comment by LKreutzer** [comment]
Yes, this is possible you can find all the information on [this page](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/changing-your-github-username). If you follow the steps here you can also check which names are available.

**Comment by ncerutti** [comment]
Hi, click on your picture on the top right, choose Settings -> Account and it will be the topmost option. Remember to update eventual links you might have to your page/repos!
